datasets/NJ_Imagery.yaml

```diff
@@ -0,0 +1,23 @@
+Name: New Jersey Statewide Digital Aerial Imagery Catalog 
+Description:  TThe New Jersey Office of GIS, NJ Office of Information Technology manages a series of 11 digital orthophotography and scanned aerial photo maps collected at various years ranging from 1930 to 2017. Each year’s worth of imagery are available as Cloud Optimized GeoTIFF (COG) files and some years are available as compressed MrSID and/or JP2 files.  Additionally, each year of imagery is organized into a tile grid scheme covering the entire geography of New Jersey.  Many years share the same tiling grid while others have unique grids as defined by the project at the time.
+Contact:  njgin@oit.nj.gov
+ManagedBy:  The New Jersey Office of GIS, NJ Office of Information Technology
+UpdateFrequency: None
+Tags:
+  - aerial imagery
+  - earth observation
+  - geospatial
+  - imaging
+  - mapping
+License:  The State of New Jersey provides the service "as is". The State makes no guarantee or warranty concerning the accuracy of information contained in the data. Also, the State makes no warranty, either expressed or implied, regarding the condition of the product or its fitness for any particular purpose. The burden for determining fitness for use lies entirely with the user. Although these data files and service have been processed successfully on computers at the State, no warranty is made by the State regarding the use of the data or service on any other system, nor does the fact of distribution constitute or imply such a warranty.
+Resources:
+  - Description: New Jersey digital orthophotography archive.  
+    ARN: arn:aws:s3:::njogis-imagery
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+	Tutorials:

```

Comment:
@brianembley looks like you've got some tabs in here which the linter does not like. If you could change these tabs to spaces, that should get past this error. It's like this in the other file as well. 

Also, please make sure to add a Documentation field for each entry as that's required as well. Thanks!

Reply:
@brianembley just wanted to ping here again in case you wanted to get this in before out next announcement.